subprojects/docs/src/snippets/java/customDirs/tests/javaCustomReportDirs.out

```diff
@@ -1,2 +1,2 @@
-my-reports
+my-reports/tests
```

Comment:
🟡 Please avoid changing the expected output unless absolutely necessary.

Reply:
Fixed, I've reverted to the original expected output in the latest PR.